src/pool_worker_master.erl

```diff
@@ -0,0 +1,49 @@
+-module(pool_worker_master).
+% using list_to_integer as command line env variable declaration looks to set string
+-define(WORKER_NUMBER, list_to_integer(os:getenv("WORKER_NUMBER", "4"))).
+
+-export([init_pool_worker_master/0, find_worker_for_machine_execution/5]).
+
+find_worker_for_machine_execution(0, 3, _WorkerPidList, _MachineConfigToExecute, ReqActorPid) ->
```

Comment:
I find the logic of this function really interesting. As we know the retry count, we could even implement backoff seamlessly!